sdk/storage/storage-internal-avro/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
We don't need this license file, since changefeed package already has one in the released package.

Reply:
Same as above. Let's keep it as it is.